### DISCUSSION TITLE
Help to deploy and test CORTX on an old PC instead of simulation/hypervisor.

### DISCUSSION THREAD

**Discussion by seasource** [Seagate/cortx#575] [Q&A] [answered]
I would like to build a system with CORTX deployed, so I could perform actual debugging and performance check instead of simulation/virtual appliance (hypervisor). Could anyone advise if this is possible with an old Intel processor based PC? I would imagine an old PC with HDDs installed should do the job? Thanks in advance for any suggestion or help provided.

**Comment by mukul-seagate11** [comment]
@mandarjoshi-stx, Do you have any idea ?

**Comment by huanghua78** [comment, marked as answer]
There are almost the same procedures to deploy CORTX in VM/Hypervisor, or to deploy CORTX in a PC/Server.

You can following the following document to do so:
https://github.com/Seagate/cortx/blob/main/doc/Cluster_Setup.md

and
https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst

and
https://github.com/Seagate/cortx-s3server/blob/main/docs/CORTX-S3%20Server%20Quick%20Start%20Guide.md

But please note, you can do "actual debugging". But if you want to do "performance check", I don't think *an old Intel processor based PC* is good enough.
